services/QuillLMS/app/services/clever_integration/sign_up/teacher.rb

```diff
@@ -13,11 +13,22 @@ def self.run(auth_hash)
     end
   end
 
+  def self.log_import(action, auth_hash)
```

Comment:
Let's add a comment `# TODO: Remove temporary logging once we collect enough samples`